pkg/vulnerability/vulnerability.go

```diff
@@ -130,6 +129,7 @@ func (c Client) getVendorSeverity(vulnID string, vuln *dbTypes.Vulnerability, so
 		return dbTypes.SeverityUnknown.String(), ""
 	}
 
+	log.Warn(fmt.Sprintf("%q and \"NVD\" don't have severity level. Severity from another vendor is used (see `VendorSeverity` in `json` format).", dataSource.Name), log.String("vulnerability-id", vulnID))
```

Comment:
Don't we need `sync.Once` to prevent from showing the same messages many times?

Reply:
I think we should show each vulnerability with non-vendor (non-nvd) severity.
It will be difficult (and possibly manual) for user to detect vulnerabilities with inappropriate severity levels if we simply say that some vulnerabilities use different severity levels.

As another way, we can save all these vulnerabilities and insert this list into one warning.
wdyt?

Reply:
right.

Reply:
I don't think 100 vulns are so many. I saw environments with 500 vulns or even more than 1000 vulns.

Reply:
Okay, let's use `once`. We can always update this - if users require it.

Update in https://github.com/aquasecurity/trivy/pull/6726/commits/b23527d5d565847b6ca941243626c5d5f7cf3ad5